connector/mqtt/vernemq/k2v-bridge/app/config.js

```diff
@@ -0,0 +1,32 @@
+const hostname = process.env.HOSTNAME || 'k2v-bridge-verne';
+const unsecuredMode = (mode) => ((mode || false) && (mode.toString().toLowerCase().trim() === 'true' || Number(mode) > 0));
+
+const app = {
+  logLevel: process.env.K2V_LOG_LEVEL || 'info',
+  baseDir: process.env.BASE_DIR || '/opt/k2v-verne',
+  hostname,
+};
+
+const mqtt = {
+  clientUsername: process.env.K2V_MQTT_USERNAME || 'k2v-bridge-verne',
+  clientId: process.env.K2V_MQTT_CLIENT_ID || hostname,
+  host: process.env.K2V_MQTT_HOST || 'vernemq-k8s',
+  port: parseInt(process.env.K2V_MQTT_PORT, 0) || 8883,
```

Comment:
`0` is an invalid value for the second parameter of the parseInt. It's working because once `0` is invalid is being used the default value which is `10`.  So, replace `0` by `10`.